scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -228,6 +230,11 @@ class MicrobeEditor{
 
     private void _addOrganelle(PlacedOrganelle@ organelle)
     {
+        // 1 - you put nucleus but you already have it
+        // 2 - you put organelle that need nucleus and you don't have it
+        if((organelle.organelle.name == "nucleus" && checkIsNucleusPresent()) || (organelle.organelle.prokaryoteChance > 0 && !checkIsNucleusPresent()))
```

Comment:
Thanks for switchinbgto the previously existing parameters. Okay, now all you need to do is  get this if statement seperated into two lines (simply have the first half on one line and the other half on the other by adding a new line after the || bit